lib/new_relic/agent/local_log_decorator.rb

```diff
@@ -5,15 +5,16 @@
 
 module NewRelic
   module Agent
-    #
     # This module contains helper methods related to decorating log messages
     module LocalLogDecorator
       extend self
 
       def decorate(message)
         return message unless decorating_enabled?
+
         metadata = NewRelic::Agent.linking_metadata
-        formatted_metadata = " NR-LINKING|#{metadata["entity.guid"]}|#{metadata["hostname"]}|#{metadata["trace.id"]}|#{metadata["span.id"]}|"
+        formatted_metadata = " NR-LINKING|#{metadata[ENTITY_GUID_KEY]}|#{metadata[HOSTNAME_KEY]}|#{metadata[TRACE_ID_KEY]}|#{metadata[SPAN_ID_KEY]}|#{escape_entity_name(metadata[ENTITY_NAME_KEY])}|"
```

Comment:
Disregard if there's a performance reason for this format, but can we do anything about this line's length? Here are two suggestions for keeping things under 120 columns at least:

```ruby
formatted_metadata = " NR-LINKING|#{metadata[ENTITY_GUID_KEY]}|#{metadata[HOSTNAME_KEY]}|" \
                     "#{metadata[TRACE_ID_KEY]}|#{metadata[SPAN_ID_KEY]}|" \
                     "#{escape_entity_name(metadata[ENTITY_NAME_KEY])}|"

formatted_metadata = [' NR-LINKING',
                      metadata[ENTITY_GUID_KEY],
                      metadata[HOSTNAME_KEY],
                      metadata[TRACE_ID_KEY],
                      metadata[SPAN_ID_KEY],
                      escape_entity_name(metadata[ENTITY_NAME_KEY])].join('|')
```

Reply:
Thanks, James! I tried the multi-line interpolation earlier, but left the `\` character inside the string, which made it appear differently. Thanks for sharing the correct syntax! It is slightly faster to have everything on one line, but it's in the range of 10-20 μs, so I think we're safe.